7/Dockerfile.rhel7

```diff
@@ -1,51 +1,42 @@
-FROM rhel7
-MAINTAINER Tom Stellard <tstellar@redhat.com>
+FROM rhscl/s2i-base-rhel7
 
-ENV SUMMARY="Red Hat LLVM Toolset 7 container image" \
-    DESCRIPTION="Platform for building applications using Red Hat LLVM Toolset 7"
+ENV LLVM_VERSION="4.0.x" \
+    SUMMARY="Platform for building projects with clang/llvm" \
+    DESCRIPTION="$SUMMARY"
 
-LABEL com.redhat.component="llvm-toolset-7-docker" \
-      name="devtools/llvm-toolset-7-rhel7" \
-      version="7" \
-      summary="$SUMMARY" \
+
+LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="LLVM Toolset 7"
-
-# To use subscription inside container yum command has to be run first (before yum-config-manager)
-# https://access.redhat.com/solutions/1443553
-RUN yum repolist > /dev/null && \
-    yum-config-manager --disable \* &> /dev/null && \
-    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
-    yum-config-manager --enable rhel-server-rhscl-7-beta-rpms && \
-    yum-config-manager --enable rhel-7-server-devtools-rpms && \
-    yum-config-manager --enable rhel-7-server-rpms && \
-    yum-config-manager --enable rhel-7-server-optional-rpms && \
-    INSTALL_PKGS="llvm-toolset-7" && \
+      io.k8s.display-name="$LLVM_VERSION" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,llvm,clang" \
+      com.redhat.component="llvm-toolset-7-docker" \
+      name="devtools/llvm-toolset-7-rhel7" \
+      version="1" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+RUN prepare-yum-repositories rhel-server-rhscl-7-rpms rhel-7-server-devtools-rpms && \
```

Comment:
I know it was present in the previous version but I am curious, is the `rhel-server-rhscl-7-rpms` repository even needed for anything in this image?